### ISSUE TITLE
a question about data formats 

### ISSUE THREAD

**Issue by rachel662** [sqjin/CellChat#308]
Hi there
I would like to use 2 available scRNA-seq datasets and use cellchat to infer cell-cell communication between cell types 
for one of the publically available datasets I am only interested in 2 of the clusters they identified 
I have a merged seurat object of the dataset and seurat objects of each dataset separately too

is it possible to run cellchat with each of the two clusters of the dataset i mention previously separately so I can see how these cell types might interact differently with the cell types in the other dataset? and if so, would you have any advice on how to go about doing it? 

cheers
Rachel

**Comment by sqjin** [comment]
@rachel662 Yes, you can run cellchat seperately on each dataset and then use the mergeCellChat function to perform comparison analysis
